.vib/spring-cloud-dataflow/cypress/cypress/integration/spring-cloud-dataflow_spec.js

```diff
@@ -0,0 +1,107 @@
+/// <reference types="cypress" />
+import { random } from './utils.js';
+import {
+  importAStreamApplication,
+  importATaskApplication,
+  createATask,
+} from './prepare-app-state.js';
+
+it('allows getting Spring Cloud Dataflow info', () => {
+  cy.visit('/dashboard');
+  cy.get('.signpost-trigger').click();
+  cy.contains('.signpost-content-body', 'Version');
```

Comment:
It would also be nice if we can check there is an actual value in the Version field.

Reply:
I've added a new method of verifying this as per your suggestion.